### PR TITLE
adds the C.L.U.W.N.E. to the clown uplink and makes simple clowns hostile

### PR DESCRIPTION
**Pull request by UselessTheremin**
### Intent of your Pull Request

these funny clowns are never used, and i think this could add a bit of spice to the round, and fits the whole style of the clown in that it's completely chaotic, either being incredible or absolutely useless. also, i made the simple clowns hostile to everyone as to not make it just a free 500 health bodyguard for the tot clown, and also not something you can just avoid instead of fighting, as youd have to start the fight.

### Why is this change good for the game?

i like these funny clowners, could be fun to see these doods run around the station

### Briefly describe your PR and the impacts of it, in layman's terms. 
adds a clown spawner grenade to the traitor uplink for 7 tc, it spawns a random clown mob that is hostile to all people

### What general grouping does this PR fall under? 
syndicate items

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
the item costs 7 telecrystals 

:cl:  
rscadd: clown spawner grenade put into uplink  
tweak: clown mobs are now innately hostile
/:cl:

### CODE REVIEW

**Comment by patpol4** [comment]
clowns are

**Review by alexkar598 — APPROVED** [review]
Code is OK.

BALANCE ABSOLUTLY ISN'T THOSE THINGS HAVE RIDICULOUS AMOUNTS OF HEALTH, RIDICULOUS AMOUNTS OF DAMAGE AND RIDICULOUS AMOUNTS OF SPEED

**Comment by UselessTheremin** [comment]
> Code is OK.
> 
> BALANCE ABSOLUTLY ISN'T THOSE THINGS HAVE RIDICULOUS AMOUNTS OF HEALTH, RIDICULOUS AMOUNTS OF DAMAGE AND RIDICULOUS AMOUNTS OF SPEED

hostile to EVERYONE

**Comment by alexkar598** [comment]
Being hostile to everyone does not justify this. Even the xenomobs who are hostile to everyone cause a shitton of damage and chaos. This is way too much.

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -1881,6 +1881,13 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 3
 	manufacturer = /datum/corporation/traitor/donkco
 	restricted_roles = list("Roboticist", "Research Director")
+	
+/datum/uplink_item/role_restricted/clowngrenade
+	name = "C.L.U.W.N.E. Grenade"
+	desc = "A tactical clown grenade which conjures a clown from the netherworlds. Be careful, it may attack you too! Not guaranteed to be useful in the slightest."
```
Comment (Hopekz) [inline code/modules/uplink/uplink_items.dm]:
```suggestion
	desc = "A tactical clown grenade which conjures a clown from the netherworlds. Be careful; it may attack you too! Not guaranteed to be useful in the slightest."
```
